src/axom/quest/Shaper.cpp

```diff
@@ -108,12 +108,50 @@ void Shaper::setVertexWeldThreshold(double threshold)
   m_vertexWeldThreshold = threshold;
 }
 
+void Shaper::setPercentError(double percent)
+{
+  using axom::utilities::clampLower;
+  using axom::utilities::clampUpper;
+  constexpr double lower = 0.;
+  constexpr double upper = 1.;
+  SLIC_WARNING_IF(
+    percent <= lower,
+    axom::fmt::format("Percent error must be greater than {}. Provided value "
+                      "was {}. Dynamic refinement will not be used.",
+                      lower,
+                      percent));
+  SLIC_WARNING_IF(percent > upper,
+                  axom::fmt::format(
+                    "Percent error must be less than {}. Provided value was {}",
+                    upper,
+                    percent));
+  if(percent <= 0.)
+  {
+    m_refinementType = RefinementUniformSegments;
+  }
+  m_percentError = clampUpper(clampLower(percent, 0.), upper);
+}
+
+void Shaper::setRefinementType(Shaper::RefinementType t)
+{
+  m_refinementType = t;
+}
+
 bool Shaper::isValidFormat(const std::string& format) const
 {
   return (format == "stl" || format == "c2c");
 }
 
 void Shaper::loadShape(const klee::Shape& shape)
+{
+  // Do not save the revolved volume in the default shaper.
+  double revolved = 0.;
+  loadShapeEx(shape, m_percentError, revolved);
+}
+
+void Shaper::loadShapeEx(const klee::Shape& shape,
```

Comment:
`loadShapeEx` -- What does the `Ex` suffix indicate? "extended"?

Reply:
I was not aware of the convention, so it was difficult for me to decipher.

Since it's a protected function, I think the name is fine, but perhaps you can add the word "execute" to the doxygen comments.

Reply:
I'm renaming it to loadShapeInternal().